### PR TITLE
ENG-1444: Improve the layout of the ECR by adopting the new wireframe

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/api/component-repository/components.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   COMPONENT_UNINSTALLATION_COMPLETED,
   COMPONENT_USAGE_LIST,
 } from 'test/mocks/component-repository/components';
-import { makeRequest, METHODS } from '@entando/apimanager';
```
Comment (gidesan) [inline src/api/component-repository/components.js:10]: please revert to `makeRequest`

File: src/api/component-repository/components.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const getECRComponent = code => (
 );
 
 export const getECRComponents = (page = { page: 1, pageSize: 10 }, params = '') => (
-  makeRequest(
+  makeMockRequest(
```
Comment (gidesan) [inline src/api/component-repository/components.js:23]: please revert to `makeRequest`

**Review by gidesan — CHANGES_REQUESTED** [review]
Looks good, we should only remove on explicit mock api call :)

File: src/api/component-repository/components.js (hunk 1 above)
Comment (gReis89) [reply src/api/component-repository/components.js]: good catch, thanks!

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/912#pullrequestreview-529115646" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**